tools/tm-signer-harness/main.go

```diff
@@ -60,6 +61,14 @@ Use "tm-signer-harness help <command>" for more information about that command.`
 		fmt.Println("")
 	}
 
+	hd, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Println("The UserHomeDir is not defined, use the default TM Home PATH \"~/.tendermint\".")
```

Comment:
```suggestion
		fmt.Println("The UserHomeDir is not defined, setting the default TM Home PATH to \"~/.tendermint\"")
```

Reply:
Also is there any value in printing out what the actual error was

Reply:
@cmwaters there is only one error returns from `os.UserHomeDir()` , `%userprofile%`/`$HOME`/`$home`(depends on OS) `is not defined`